dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -10,13 +10,17 @@ RUN apt-get update
 
 RUN apt-get install -f -y ifupdown arping libdbus-1-3 libdaemon0 libjansson4
 
-RUN apt-get install -f -y ndisc6
+RUN apt-get install -f -y ndisc6 tcpdump python-scapy
```

Comment:
>tcpdump [](start = 33, length = 7)

why do we need tcpdump? is it for debugging purpose? if yes, then we should remove it.

Reply:
Two reasons:
1, scapy requires tcpdump for some functions though we have not used it now.
2, tcpdump is good tool to have in vs docker and in swss docker for testing.  It is a basic tool that is good/required even for production.

Reply:
right now we have both tcpdump installed in the base image and in this docker, do we want to keep only one copy?